rasa/core/training/story_conflict.py

```diff
@@ -1,13 +1,21 @@
 from collections import defaultdict
 import logging
-from typing import Dict, Generator, List, NamedTuple, Optional, Text, Tuple
+import json
+from typing import Dict, Generator, List, NamedTuple, Optional, Text, Tuple, Any
 
 from rasa.core.featurizers.tracker_featurizers import MaxHistoryTrackerFeaturizer
 from rasa.shared.core.constants import ACTION_LISTEN_NAME, PREVIOUS_ACTION, USER
 from rasa.shared.core.domain import Domain, PREV_PREFIX, State, SubState
 from rasa.shared.core.events import ActionExecuted, Event
 from rasa.shared.core.generator import TrackerWithCachedStates
-from rasa.shared.nlu.constants import INTENT
+
+from rasa.nlu.model import Trainer
+from rasa.nlu.components import Component
+from rasa.nlu.tokenizers.tokenizer import Tokenizer
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.shared.nlu.constants import TEXT
+from rasa.shared.nlu.training_data.message import Message
+from rasa.shared.utils.io import raise_warning
```

Comment:
please import the module instead of the function directly.

Reply:
Damn! I always overlook those! Sorry :/